components/brave_welcome_ui/containers/screens/footerBox.tsx

```diff
@@ -13,7 +13,7 @@ import {
   SkipButton,
   FooterButton,
   Bullet
-} from 'brave-ui/features/welcome'
+} from '../../../../components/brave_welcome_ui/components'
```

Comment:
nit: exits UI folder then goes back in - can we replace `../../../../components/brave_welcome_ui` with `../../`?